aghhh.py

```
@@ -0,0 +1,108 @@
+    elif bmi < 25:
+        return "Normal weight"
+    elif bmi < 30:
+        return "Overweight"
+    else:
+        return "Obese"
+
+print(calculate_bmi(70, 0))  # Division by zero error
+
+# Class with poor design and functionality
+class Student:
+    def __init__(self, name, grades=[]):  # Mutable default argument
```

Ruff (0.8.2)

Do not use mutable data structures for argument defaults

Replace with `None`; initialize within function

(B006)